pipelines/apDetectorVisitQualityCore.yaml

```diff
@@ -105,12 +105,20 @@ tasks:
       atools.interpolatedTemplatePsfSize.produce.plot.zAxisLabel: pixels
       python: |
         from lsst.analysis.tools.atools import DiffimSpatialMetricsHistPlot, DiffimSpatialMetricsInterpolatePlot
+  initialPviCore:
+    class: lsst.analysis.tools.tasks.CalexpSummaryAnalysisTask
+    config:
+      atools.calexpSummaryMetrics: CalexpSummaryMetrics
```

Comment:
Since we call these PVIs now, consider renaming everything that references a `calexp` here. You probably should leave `CalexpSummaryAnalysisTask` as-is since I imagine that is used elsewhere, but both names in `atools.calexpSummaryMetrics: CalexpSummaryMetrics` should change.

Reply:
We don't need to duplicate either the metric or the task, actually--retargeting the `outputName` below is sufficient that all of the downstream tools only see `pvi`, not `calexp`.  Since the underlying `summaryStats` input is the same in both cases, we avoid unnecessary code duplication.

Reply:
OK, but at least rename `atools.calexpSummaryMetrics` to `pviSummaryMetrics`. That name does make it through to the final metrics .json file, and is how it will appear in chronograf.

Reply:
Ah, I see.  I didn't realize the dataset type didn't reflect what was in the .json payload.  Confirmed the fix (I called it `initialPviSummaryMetrics`).